etc/adminhtml/system.xml

```diff
@@ -94,6 +94,21 @@
                         <config_path>payment/omise_offsite_alipay/title</config_path>
                     </field>
                 </group>
+                <group id="omise_offsite_citipoints" translate="label" sortOrder="160" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Enable Citi Points Solution</label>
+                    <comment>Enables customers to pay using Citi Points method</comment>
+                    <field id="active" translate="label comment" type="select" sortOrder="170" showInDefault="1" showInWebsite="1" showInStore="1">
+                        <label>Enable/Disable</label>
```

Comment:
The label 'enable/disable' makes no sense with the options 'yes' and 'no'. 'Enabled?' would be a better choice

Reply:
thanks... so what better messages would you recommend.
just notice that it is all payment methods have similar texts, so I will change in separated PR all of them.

I would leave enable/disable but change title. or what do you think?

Reply:
I changed this texts here: #232